README.md

```diff
@@ -213,31 +213,32 @@ if fail_count > 0:
 
 ##5. Searching Images
```

Comment:
Can you change the title to 5.Solutions and update the table of content as well?